controller/src/beerocks/cli/beerocks_cli_bml.cpp

```diff
@@ -2144,6 +2205,104 @@ int cli_bml::get_dcs_scan_results(const std::string &radio_mac, uint32_t max_res
     return 0;
 }
 
+/**
+ * get all client list.
+ * 
+ * @return 0 on success.
+ */
+int cli_bml::client_get_client_list()
+{
+    char client_list[256];
+    unsigned int client_list_size;
+
+    int ret = bml_client_get_client_list(ctx, client_list, &client_list_size);
+    auto client_list_vec =
+        beerocks::string_utils::str_split(std::string(client_list, client_list_size), ',');
+
+    std::cout << "client list:" << std::endl;
+    for (const auto &client : client_list_vec) {
+        std::cout << "- " << client << std::endl;
```

Comment:
if you use `std::endl` there's no added value with the `- ` IMO
(nitpick)

Reply:
Aesthetics